### DISCUSSION TITLE
Show buttons on mobile

### DISCUSSION THREAD

**Discussion by t4px** [biati-digital/glightbox#448] [General]
Hello
How can I show prev - next buttons on mobile dimensions?

Now looks like buttons are disabled?

![Screenshot_10](https://github.com/biati-digital/glightbox/assets/146641374/ffbc2989-cfca-4b99-bea7-2d4323ee25f1)

**Comment by gingerchew** [comment]
The buttons are just hidden with some CSS, if you inspect it, you can find what rule needs to be overridden
